sdks/python/apache_beam/examples/streaming_wordcount.py

```diff
@@ -46,8 +48,10 @@ def run(argv=None):
       '--output_topic', required=True,
       help='Output PubSub topic of the form "/topics/<PROJECT>/<TOPIC>".')
```

Comment:
Changed it to the current PubSub topic format.  The Dataflow service accepts both the old (v1beta1) and new formats.